XMonad/Layout/Fullscreen.hs

```diff
@@ -89,6 +89,7 @@ fullscreenSupport c = c {
 -- of the above have been sent.
 data FullscreenMessage = AddFullscreen Window
                        | RemoveFullscreen Window
+                       | ToggleFullscreen Window
```

Comment:
I feel like the above documentation should be updated somehow. It seems that the existing messages are meant to be sent by the manage and event hooks, while the new `ToggleFullscreen` message is intended to be sent manually, so this may need a bit more explanation than usual. Also, is one supposed to send `FullscreenChanged` after sending `ToggleFullscreen`?

(Disclaimer: I'm not a user of X.L.Fullscreen, I only use `fullscreenEventHook` from `X.H.EwmhDesktops`.)

Reply:
Thx for your feedback! Honestly I am very new to XMonad so I'm not yet that familiar with how things are supposed to be used but I think you are right.

> Also, is one supposed to send FullscreenChanged after sending ToggleFullscreen?

I haven't thought of that but probably yes i guess? If so then maybe it would be better to expose a function like toggleFullscreen that sends both the ToggleFullscreen and FullscreenChanged messages and just for completeness also an addFullscreen and removeFullscreen function.
Then only these functions are supposed to be used manually